recipes/r-pqlseq/meta.yaml

```diff
@@ -0,0 +1,99 @@
+{% set version = '1.2.1' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-pqlseq
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/PQLseq_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/PQLseq/PQLseq_{{ version }}.tar.gz
+  sha256: c03c43144309913ebbceb47609456538df6e571040149edd7f31d8b3f7b7de53
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ stdlib('c') }}                # [not win]
+    - {{ stdlib('m2w64_c') }}          # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+    - r-matrix                     # [build_platform != target_platform]
+    - r-rcpp >=0.12.14             # [build_platform != target_platform]
+    - r-rcpparmadillo              # [build_platform != target_platform]
+    - r-doparallel                 # [build_platform != target_platform]
+    - r-foreach                    # [build_platform != target_platform]
+  host:
+    - libblas
+    - liblapack
+    - libgomp                      # [not osx]
+    - llvm-openmp                  # [osx]
```

Comment:
`libgomp  # [not osx]` should be moved to `build` (remove from `host`).

`llvm-openmp  # [osx]` should be in all three (`build`, `host`, `run`), otherwise it leads to bad run constraints.